infra/lib/auth/stack.create-user.ts

```diff
@@ -24,8 +24,8 @@ interface CreateUserResponse {
   error?: string;
 }
 
-const USER_POOL_ID = process.env.USER_POOL_ID;
-const CLIENT = new cognito.CognitoIdentityProviderClient({
+const { USER_POOL_ID } = process.env;
+const CLIENT = new cognito.CognitoIdentityProvider({
   region: process.env.REGION,
 });
```

Comment:
```suggestion
const FULL_CLIENT = new cognito.CognitoIdentityProvider({
  region: process.env.REGION,
});
const CLIENT = new cognito.CognitoIdentityProviderClient({
  region: process.env.REGION,
});
```

Was seeing failures trying to deploy the new resources for integ testing. By using the `CognitoIdentityProvider` constructor on only the MFA section, I could get a good deploy.